docs/plugin/custom-executor.md

```diff
@@ -77,10 +78,29 @@ For a final implementation, see the [Botkube template repository](./quick-start.
 
    ```go
    // Metadata returns details about the Echo plugin.
-   func (EchoExecutor) Metadata(context.Context) (api.MetadataOutput, error) {
+   func (*EchoExecutor) Metadata(context.Context) (api.MetadataOutput, error) {
      return api.MetadataOutput{
        Version:     "1.0.0",
        Description: "Echo sends back the command that was specified.",
+       JSONSchema: api.JSONSchema{
+       Value: heredoc.Doc(`{
+          "$schema": "http://json-schema.org/draft-04/schema#",
+          "title": "botkube/echo",
+          "description": "example echo plugin",
+          "type": "object",
+          "properties": {
+            "formatOptions": {
+              "description": "options to format echoed string",
+              "type": "array",
+              "items": [
+                "type": "string",
+                "enum": ["bold", "italic"]
+              ]
+            }
+          },
+          "required": []
+        }`),
+       },
```

Comment:
Please use a real JSON schema:
https://github.com/kubeshop/botkube/blob/main/cmd/executor/echo/main.go#LL73C19-L73C19

Reply:
Consistency: the example uses `formatOptions: [italic, bold]` . Using `changeResponseToUpperCase` can create confusion.

Reply:
Ah, you're right, sorry. Good catch.

In this case could you please bring back the previous schema? The previous one was invalid, but I fixed it so you can copy and paste the following one:

```json
{
  "$schema": "http://json-schema.org/draft-04/schema#",
  "title": "botkube/echo",
  "description": "example echo plugin",
  "type": "object",
  "properties": {
    "formatOptions": {
      "description": "options to format echoed string",
      "type": "array",
      "items": {
        "type": "string",
        "enum": [
          "bold",
          "italic"
        ]
      }
    }
  },
  "additionalProperties": false,
  "required": []
}
```

Thanks!